docs/source/reference/kubernetes/kubernetes-setup.rst

```diff
@@ -274,8 +284,47 @@ All SkyPilot tasks are run in pods inside a Kubernetes cluster. As a cluster adm
 you can inspect running pods (e.g., with :code:`kubectl get pods -n namespace`) to check which
 tasks are running and how many resources they are consuming on the cluster.
 
-Additionally, you can also deploy tools such as the `Kubernetes dashboard <https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/>`_ for easily viewing and managing
-SkyPilot tasks running on your cluster.
+Below, we provide tips on how to monitor SkyPilot resources on your Kubernetes cluster.
+
+.. _kubernetes-observability-skystatus:
+
+SkyPilot CLI
+^^^^^^^^^^^^
+
+We provide a convenience command, :code:`sky status --kubernetes`, to view the status of all SkyPilot resources in the cluster.
```

Comment:
Could we replace --kubernetes with --k8s? People will use whatever the docs recommend.